webui/react/vite.config.mts

```diff
@@ -164,7 +164,11 @@ export default defineConfig(({ mode }) => ({
     coverage: {
       ...configDefaults.coverage,
       include: ['src'],
-      exclude: [...configDefaults.coverage.exclude, 'src/vendor/**/*', 'src/services/api-ts-sdk/*']
+      exclude: [
+        ...(configDefaults.coverage.exclude ?? []),
```

Comment:
`configDefaults.coverage.exclude` type is `BaseCoverageOptions.exclude?: string[] | undefined`, so cover `undefined` case